scripts/karma-rollup-plugin.js

```diff
@@ -125,18 +120,31 @@ function bundlePreprocessor(config) {
 
     const bundle = await rollup.rollup({
       input: fileMap.get(file.path),
-      plugins: [...plugins, multiEntry({exports: false})]
+      plugins: [...plugins, multiEntry({exports: false})],
+      external: ['sinon'],
+      onwarn: (warning, warn) => {
```

Comment:
looks like we could just copy the `onwarn` from `rollup.config.js` since we already require it, right?

Reply:
We could indeed. I'll make that change

Reply:
Fixed in e5fd667d